diversity.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: Diversity
+permalink: /diversity/
+isStaticPost: true
+---
+
+##### DevFest Berlin supports diversity in technology
+
+We recognize the current status of our industry and we want to want to change it.
+
+Our goal is to enable everybody to learn and teach programming. As such, we
+want to enable underrepresented groups in technology to feel welcomed and help
+increase the diversity in technology and science.
+
+To this end, we have have a couple of ways to support our mission:
+
+- we will have a number of tickets for which only underrepresented groups can apply
+- we will have a mentorship program for new programmers
+
+The mentorship program will pair developers willing to share their passion and
```

Comment:
What is actually happening at the mentorship program is not clear if you are not familiar with the concept. How about -

`The mentorship program will pair an awardee with a developer mentor to help familiarize the awardee with the conference. We believe creating a dialogue between developers at different levels will yield a great value both to the mentor and the awardee and hope everyone will have a great opportunity to learn from it.`